webapp/src/components/AuctionPage/AuctionPage.js

```diff
@@ -0,0 +1,123 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { Message, Header, Grid, Container } from 'semantic-ui-react'
+import { utils } from 'decentraland-commons'
+
+import { parcelType } from 'components/types'
+import ParcelPreview from 'components/ParcelPreview'
+import ParcelAttributes from 'components/ParcelAttributes'
+import { t } from '@dapps/modules/translation/utils'
+
+import './AuctionPage.css'
+
+// TODO: Real number here
+const MAX_PARCELS_PER_TX = Infinity
+
+export default class AuctionPage extends React.PureComponent {
+  static propTypes = {
+    allParcels: PropTypes.objectOf(parcelType)
+  }
+
+  constructor(props) {
+    super(props)
+    this.state = {
+      selectedCoordsById: {}
+    }
+  }
+
+  selectUnownedParcel = (asset, { x, y }) => {
+    const { selectedCoordsById } = this.state
+
+    // TODO: Check ownership with contract
+    if (asset.owner != null) return
+
+    let newSelectedCoordsById = {}
+
+    const isSelected = selectedCoordsById[asset.id] !== undefined
+    if (isSelected) {
+      newSelectedCoordsById = utils.omit(selectedCoordsById, asset.id)
+    } else {
+      newSelectedCoordsById = {
+        ...selectedCoordsById,
+        [asset.id]: { x, y }
+      }
+    }
+
+    if (this.hasReachedAddLimit(newSelectedCoordsById)) return
```

Comment:
`undefined is not a function` (:

this should be `hasReachedLimit` or the function below should be named `hasReachedAddLimit`

Reply:
dang, I changed in another PR and not here, good catch!